src/linter/phpdoc_util.go

```diff
@@ -68,6 +68,7 @@ func parseClassPHPDocMethod(ctx *rootContext, result *classPhpDocParseResult, pa
 		Flags:        funcFlags,
 		MinParamsCnt: 0, // TODO: parse signature and assign a proper value
 		AccessLevel:  meta.Public,
+		FromDoc:      true,
```

Comment:
I believe it's more efficient to add another flag to a Func/Property.
I'm not sure that we have a `Flags` field for properties, but we definitely have flags for functions and classes.